classes/class-wc-connect-service-settings-store.php

```diff
@@ -110,6 +110,40 @@ public function can_user_manage_payment_methods() {
 				( is_a( $master_user, 'WP_User' ) && $current_user->ID === $master_user->ID );
 		}
 
+		/**
+		 * Gets the store's location settings.
+		 *
+		 * @return array
+		 */
+		public function get_endshipper_address() {
+			$email       = '';
+			$master_user = WC_Connect_Jetpack::get_master_user();
+
+			if ( is_a( $master_user, 'WP_User' ) ) {
+
+				$connected_data = WC_Connect_Jetpack::get_connected_user_data( $master_user->ID );
+
+				if ( is_array( $connected_data ) && ! empty( $connected_data['email'] ) ) {
+					$email = $connected_data['email'];
+				}
+			}
+
+			$store_settings = array(
+				'company'   => html_entity_decode( get_bloginfo( 'name' ), ENT_QUOTES ), // HTML entities may be saved in the option.
+				'address_1' => WC()->countries->get_base_address(),
+				'address_2' => WC()->countries->get_base_address_2(),
+				'city'      => WC()->countries->get_base_city(),
+				'state'     => WC()->countries->get_base_state(),
+				'country'   => WC()->countries->get_base_country(),
+				'postcode'  => WC()->countries->get_base_postcode(),
+				'email'     => $email,
+				'phone'     => '',
+				'name'      => '',
+			);
+
+			return apply_filters( 'endshipper_store_settings', $store_settings, array() );
```

Comment:
I think this was accidentally left over.

```suggestion
			return apply_filters( 'endshipper_store_settings', $store_settings );
```